model/cloud.go

```diff
@@ -257,3 +257,16 @@ type ProductLimits struct {
 	Messages     *MessagesLimits     `json:"messages,omitempty"`
 	Teams        *TeamsLimits        `json:"teams,omitempty"`
 }
+
+type BootstrapSelfHostedSignupRequest struct {
+	Email string `json:"email"`
+}
+
+type BootstrapSelfHostedSignupResponse struct {
+	Progress string `json:"progress"`
+}
+
+type BootstrapSelfHostedSignupResponseInternal struct {
+	Progress string `json:"progress"`
+	License  string `json:"license"`
+}
```

Comment:
Should this be under `cloud.go` if we are explicitly separating from cloud endpoints?

Reply:
Saw further down that we are still going through the enterprise `Cloud` interfaces to talk to CWS